### DISCUSSION TITLE
Not able to compile with dirty working tree with latest Tycho version

### DISCUSSION THREAD

**Discussion by Omcsesz** [eclipse-tycho/tycho#864] [Q&A]
Hello, Team!

I'm trying to build a project which uses the latest version of Tycho. During build, I get the following error message:
```

[ERROR] Failed to execute goal org.eclipse.tycho:tycho-packaging-plugin:2.7.0:build-qualifier-aggregator (default-build-qualifier-aggregator) on project org.eclipse.tm4e.feature: Working tree is dirty.
You are trying to use tycho-buildtimestamp-jgit on a directory that has uncommitted changes (see details above).
[ERROR] Either commit all changes/add files to .gitignore, or enable fallback to default timestamp provider by configuring jgit.dirtyWorkingTree=warning for tycho-packaging-plugin
```

I'm using the latest version of Eclipse. 

Can somebody please help me with this?

**Comment by laeubi** [comment]
> I'm using the latest version of Eclipse.

You mean Tycho?

> You are trying to use tycho-buildtimestamp-jgit on a directory that has uncommitted changes (see details above).

You should see the offending files in the log above this message, you have the following options:

1. commit all changes
2. add files to .gitignore
3. configuring jgit.dirtyWorkingTree=warning for tycho-packaging-plugin

as indicated in the error message.

**Reply by Omcsesz** [reply]
> > I'm using the latest version of Eclipse.
> 
> You mean Tycho?
> 
I mean both of them.
